gen/generator.jl

```diff
@@ -0,0 +1,16 @@
+using Clang
+using Clang.Generators
+using XGBoost_jll
```

Comment:
Could you add a comment in this file as to what it does? Or maybe a README file at `gen/README.md` with some pointers to Clan generators documentation?

Reply:
I added a link